### DISCUSSION TITLE
Version available for Macbook M2

### DISCUSSION THREAD

**Discussion by HLC1425** [artisan-roaster-scope/artisan#1013] [Q&A] [answered]
Is there a version that is available on Macbook with M2 chip?

**Comment by MAKOMO** [comment, marked as answer]
Current version runs fine under Rosetta-II on M1- and M2-based macOS machines. There is no native build for M1/M2 processors of Artisan and there will most likely be none before 2024 or so.

**Reply by HLC1425** [reply]
Thanks for your quick reply!
